src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java

```diff
@@ -172,7 +175,7 @@ public boolean get(URIish uri, CredentialItem... items) throws UnsupportedCreden
 
             @Override
             public char[] getPassphrase(URIish uri, int attempt) {
-                log.debug("getPassphrase: {}, attempt: {}", uri.toString(), attempt);
+                log.debug("getPassphrase: {}, attempt: {}", uri, attempt);
```

Comment:
I just saw that. Do we really want that the passphrase is logged?

Reply:
The passphrase wasn’t logged, but the URI. The original intention of the logging seems to have been to show with which arguments the method `getPassprase()` was called.
I assume this is a relict from whenever this code was originally created and is no longer a useful debug log. Therefore, I removed it entirely.